Framework/Source/Graphics/Material/Material.cpp

```diff
@@ -139,14 +139,14 @@ namespace Falcor
         updateEmissiveType();
     }
 
-    void Material::setDiffuseColor(const vec4& color)
+    void Material::setBaseColor(const vec4& color)
     {
-        mParamBlockDirty = mParamBlockDirty || (mData.diffuse != color);
-        mData.diffuse = color;
+        mParamBlockDirty = mParamBlockDirty || (mData.baseColor != color);
+        mData.baseColor = color;
         updateDiffuseType();
     }
 
-    void Material::setSpecularColor(const vec4& color)
+    void Material::setSpecularParams(const vec4& color)
```

Comment:
Why call it specular if it's not specular? It's AO, metal, rough - find a good name for it.